src/Core/Filesystem.re

```diff
@@ -215,10 +215,13 @@ let rmdir = path =>
 
 let unsafeFindHome = () =>
```

Comment:
Since I'm in the area, does anyone have a smarter name for this/think I should change this?

Its not really a home folder on Windows, its the config location. Its also only used in the `getOniDirectory`, so could be moved into there and given a smarter name (since its really used to get `XDG_CONFIG_DIR`), which would prevent us using it incorrectly in the future.

Reply:
What about `getUserDataDirectory`?

But also, `unsafe` is incorrect, as it typically means undefined behaviour. The convention for a function that raise exceptions is an `Exn` suffix (or `_exn` in OCaml).